### DISCUSSION TITLE
Command click in the file tree should open the file in a new tab

### DISCUSSION THREAD

**Discussion by ibash** [community/community#42513] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Related to https://github.com/community/community/discussions/15809. Command-clicking in the file tree should open the file in a new tab.

**Comment by Jackenmen** [comment]
The same goes for `Ctrl+click` on non-macOS platforms, as well as for the middle mouse button click. The "Open in new tab" option should also be present in the context menu when you right-click on the entry in the file tree. Basically, the clickable `div`s should be replaced with actual links.
